core/ledger/kvledger/txmgmt/validation/types.go

```diff
@@ -8,59 +8,46 @@ package validation
 
 import (
 	"github.com/hyperledger/fabric-protos-go/peer"
-	"github.com/hyperledger/fabric/common/flogging"
 	"github.com/hyperledger/fabric/core/ledger/internal/version"
 	"github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/privacyenabledstate"
 	"github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/rwsetutil"
 )
 
-var logger = flogging.MustGetLogger("validation")
-
-// Block is used to used to hold the information from its proto format to a structure
+// block is used to used to hold the information from its proto format to a structure
 // that is more suitable/friendly for validation
-type Block struct {
-	Num uint64
-	Txs []*Transaction
+type block struct {
+	num uint64
+	txs []*transaction
 }
 
-// Transaction is used to hold the information from its proto format to a structure
+// transaction is used to hold the information from its proto format to a structure
 // that is more suitable/friendly for validation
-type Transaction struct {
-	IndexInBlock            int
-	ID                      string
-	RWSet                   *rwsetutil.TxRwSet
-	ValidationCode          peer.TxValidationCode
-	ContainsPostOrderWrites bool
+type transaction struct {
+	indexInBlock            int
+	id                      string
+	rwset                   *rwsetutil.TxRwSet
+	validationCode          peer.TxValidationCode
+	containsPostOrderWrites bool
 }
 
-// PubAndHashUpdates encapsulates public and hash updates. The intended use of this to hold the updates
+// publicAndHashUpdates encapsulates public and hash updates. The intended use of this to hold the updates
 // that are to be applied to the statedb  as a result of the block commit
-type PubAndHashUpdates struct {
-	PubUpdates  *privacyenabledstate.PubUpdateBatch
-	HashUpdates *privacyenabledstate.HashedUpdateBatch
-}
-
-// ErrPvtdataHashMissmatch is to be thrown if the hash of a collection present in the public read-write set
-// does not match with the corresponding pvt data  supplied with the block for validation
-type ErrPvtdataHashMissmatch struct {
-	Msg string
-}
-
-func (e *ErrPvtdataHashMissmatch) Error() string {
-	return e.Msg
+type publicAndHashUpdates struct {
+	publicUpdates *privacyenabledstate.PubUpdateBatch
```

Comment:
When I refactor `privacyenabledstate`, I will update the name of this struct too.